openapi.yaml

```diff
@@ -1,190 +1,432 @@
     url: http://www.gnu.org/licenses/agpl-3.0.en.html
   contact:
-    name: IS Data Architecture Team
-    url: https://is.oregonstate.edu/data-architecture
-    email: isdataarchitecture@oregonstate.edu
+    name: David Kalcic
+    email: kalcicd@oregonstate.edu
 schemes:
   - https
 host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/parks-api
+consumes:
+  - application/json
 produces:
   - application/json
 paths:
-  /pets:
+  /parks:
     get:
-      summary: List all pets
+      summary: List all parks
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - parks
+      description: Get all parks
+      operationId: getParks
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: amenities
+          description: Filter by park amenities (play area, hiking trails, etc.)
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - ballfield
+              - barbeque grills
+              - basketball courts
+              - bike paths
+              - boat ramps
+              - dogs allowed
+              - drinking water
+              - fishing
+              - hiking trails
+              - horseshoes
+              - natural area
+              - offleash dog park
+              - open fields
+              - picnic shelters
+              - picnic tables
+              - play area
+              - restrooms
+              - scenic view point
+              - soccer fields
+              - tennis courts
+              - volleyball
+        - in: query
+          name: parkName
+          description: Filter parks by name
+          required: false
+          type: string
+        - in: query
+          name: ownerId
+          description: Filter parks by owner ID
+          required: false
+          type: string
+        - in: query
+          name: city
+          description: Filter parks by city
+          required: false
+          type: string
+        - in: query
+          name: state
+          description: Filter parks by state
+          required: false
+          type: string
+        - in: query
+          name: zipCode
+          description: Filter parks by zip code
           required: false
           type: string
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/ParksResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
```

Comment:
What would the response for this endpoint be if I query with an incorrect query parameter value such as `GET /parks?amenities=false`?